src/runtime/server/middleware/session/index.ts

```diff
@@ -10,7 +10,7 @@ import { useRuntimeConfig } from '#imports'
 const SESSION_COOKIE_NAME = 'sessionId'
 const safeSetCookie = (event: H3Event, name: string, value: string, createdAt: Date) => {
   const sessionOptions = useRuntimeConfig().session.session as SessionOptions
-  const expirationDate = sessionOptions.expiryInSeconds
+  const expirationDate = sessionOptions.expiryInSeconds !== false
```

Comment:
@BracketJohn We should also check here for explicit boolean/false value

Reply:
yes, good catch